spec/acceptance/bundle_management_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper_acceptance'
+require 'fileutils'
+
+describe 'Managing Gemfile dependencies' do
+  # Use a absolute path for the target directory, to avoid depending on test environment differences below.
+  # Note that the parent directory of test_module has to exist.
+  # Note that this has to be in the top-scope here, to be available for the `describe file` below.
+  target_dir = Gem.win_platform? ? 'C:/test_module' : '/tmp/test_module'
+
+  before(:all) do
+    # We need a module to test inside of.
+    shell_ex("#{path_to_pdk} new module test_module --skip-interview #{target_dir}")
+  end
+
+  after(:all) do
+    shell_ex("rm -rf #{target_dir}")
+  end
+
+  context 'when there is no Gemfile.lock' do
+    before(:all) do
+      shell_ex("rm -f #{target_dir}/Gemfile.lock")
+      # TODO: come up with a way to invoke only the bundler stuff without trying to run unit tests
+      @result = shell_ex("#{path_to_pdk} test unit", chdir: target_dir)
+    end
+
+    it "checks for missing Gemfile dependencies" do
+      expect(@result).to have_attributes(exit_code: 0, stderr: match(/Checking for missing Gemfile dependencies/i) )
+    end
+
+    describe file("#{target_dir}/Gemfile.lock") do
+      it { is_expected.to be_file }
+    end
+  end
+
+  context 'when there is an invalid Gemfile' do
+    before(:all) do
+      shell_ex("mv #{target_dir}/Gemfile #{target_dir}/Gemfile.old")
```

Comment:
Are there platform independent implementations for `rm` and `mv` available yet or do we need to add those?